server/routes/search/handlers/caSearch.ts

```diff
@@ -26,6 +30,14 @@ export default class CaSearch {
     const { user } = res.locals
     const { caseloadsSelected = [] } = req.session
 
+    const hasMultipleCaseloadsInNomis = user.prisonCaseload.length > 1
+    const hasSelectedMultiplePrisonCaseloads = caseloadsSelected.length > 1
```

Comment:
this seems to be duplicated across quite a few handlers and adds complexity to tests where it should probably just be done centrally in populatecurrentuser.